### PR TITLE
Support `INSAR_ISCE_BURST` job type

### PR DESCRIPTION
**Pull request by jtherrmann**
I named the new functions `submit_insar_isce_burst_job` and `prepare_insar_isce_burst_job` to match the job type `INSAR_ISCE_BURST`, but I'm open to a different naming scheme.

TODO:

- [x] Wait to merge until `INSAR_ISCE_BURST` job type is available in production HyP3.
- [x] Update version number for Changelog entry.

### CODE REVIEW

File: src/hyp3_sdk/hyp3.py [hunk 1]
```diff
@@ -411,6 +411,62 @@ def prepare_insar_job(cls,
...
+        return self.submit_prepared_jobs(prepared_jobs=job_dict)
+
+    @classmethod
+    def prepare_insar_isce_burst_job(cls,
+                                     granule1: str,
+                                     granule2: str,
+                                     name: Optional[str] = None,
+                                     apply_water_mask: bool = False,
+                                     looks: Literal['20x4', '10x2', '5x1'] = '20x4') -> dict:
+        """Submit an InSAR ISCE burst job.
```
Comment (cirrusasf) [inline src/hyp3_sdk/hyp3.py:445]: suggest change the description: "Prepare an INSAR ISCE burst job"

File: src/hyp3_sdk/hyp3.py [hunk 2]
```diff
@@ -411,6 +411,62 @@ def prepare_insar_job(cls,
...
+
+        Args:
+            granule1: The first granule (scene) to use
+            granule2: The second granule (scene) to use
+            name: A name for the job
+            apply_water_mask: Set pixels over coastal and large inland water bodies to NoData
+                in wrapped and unwrapped phase GeoTIFFs (after unwrapping)
+            looks: Number of looks to take in range and azimuth
```
Comment (cirrusasf) [inline src/hyp3_sdk/hyp3.py:453]: suggestion: Number of looks to take in 'range_numberxazimuth_number' format, the purpose is to make sure user input the correct format of the looks argument

File: src/hyp3_sdk/hyp3.py [hunk 3]
```diff
@@ -411,6 +411,62 @@ def prepare_insar_job(cls,
...
+                                    looks: Literal['20x4', '10x2', '5x1'] = '20x4') -> Batch:
+        """Submit an InSAR ISCE burst job.
+
+        Args:
+            granule1: The first granule (scene) to use
+            granule2: The second granule (scene) to use
+            name: A name for the job
+            apply_water_mask: Set pixels over coastal and large inland water bodies to NoData
+                in wrapped and unwrapped phase GeoTIFFs (after unwrapping)
+            looks: Number of looks to take in range and azimuth
```
Comment (cirrusasf) [inline src/hyp3_sdk/hyp3.py:428]: suggestion: Number of looks to take in 'range_numberxazimuth_number' format

File: src/hyp3_sdk/hyp3.py (hunk 1 above)
Comment (jtherrmann) [inline src/hyp3_sdk/hyp3.py:445]:
```suggestion
        """Prepare an InSAR ISCE burst job.
```

**Review by cirrusasf — DISMISSED** [review]

**Comment by jtherrmann** [comment]
Successful test job: https://hyp3-enterprise-test.asf.alaska.edu/jobs/41161733-2df0-427f-a00f-8f7059be568a

**Comment by forrestfwilliams** [comment]
This is ready to go now that bursts are in production

**Review by forrestfwilliams — APPROVED** [review]
LGTM

**Pull request merged**